kernel/arch/dreamcast/hardware/network/broadband_adapter.c

```diff
@@ -18,31 +18,25 @@
 #include <dc/flashrom.h>
 #include <arch/irq.h>
 #include <arch/cache.h>
+#include <arch/memory.h>
 #include <kos/net.h>
 #include <kos/thread.h>
 #include <kos/sem.h>
 
-//#define vid_border_color(r, g, b) (void)0 /* nothing */
-
 /* Configuration definitions */
 
 #define RTL_MEM                 (0x1840000)
 
-#define RX_NOWRAP               1 /* 1 for no wrapping or 0 to use wrapping mode (Ignored for 64Kb buffer length) */
-#define RX_MAX_DMA_BURST        6 /* 2^(4+n) bytes from 0-6 (16b - 1Kb) or 7 for unlimited */
-#define RX_BUFFER_LEN_SHIFT     1 /* 0 : 8Kb, 1 : 16Kb, 2 : 32Kb, 3 : 64Kb */
-#define RX_FIFO_THRESHOLD       0 /* 2^(4+n) bytes from 0-6 (16b - 1Kb) or 7 for none */
-#define RX_EARLY_THRESHOLD      0 /* Early RX Threshold multiplier n/16 or 0 for none */
+#define RX_BUFFER_SHIFT           1 /* 0 : 8Kb, 1 : 16Kb, 2 : 32Kb, 3 : 64Kb */
```

Comment:
```suggestion
#define RX_BUFFER_SHIFT          1 /* 0 : 8Kb, 1 : 16Kb, 2 : 32Kb, 3 : 64Kb */
```
I think the alignment is off by 1 character?